README.md

```diff
@@ -1,181 +1,248 @@
+Notably, the AST eliminates any ambiguous nodes.
+For instance, Oxc distinguishes between `BindingIdentifier`, `IdentifierReference`, and `IdentifierName` instead of using a generic [estree] `Identifier`.
+These distinctions greatly improve the experience for developers who have previously worked with [estree].
+
+#### 🏆 Parser Performance
+
+Our [benchmark][parser-benchmark] reveals that the Oxc parser surpasses the speed of the [swc] parser by approximately 2 times and the [Rome] parser by 3 times.
+
+<details>
+  <summary>How is it so fast?</summary>
+  <ul>
+    <li>AST is allocated in a memory arena ([bumpalo]) for fast AST memory allocation and deallocation</li>
+    <li>Short strings are inlined by [CompactString]</li>
+    <li>No other heap allocations are done except the above two</li>
+    <li>Scope binding, symbol resolution and complicated syntax errors are not done in the parser, they are delegated to the semantic analyzer</li>
+  </ul>
+</details>
+
+### 🔸 Linter
+
+The linter embraces convention over configuration, eliminating the need for extensive configuration and plugin setup.
+Unlike other linters like [ESLint], which often require intricate configurations and plugin installations (e.g. [@typescript-eslint]),
+our linter only requires a single command that you can immediately run on your codebase:
+
+```bash
+npx oxlint@latest .
 ```
-Test262 Summary:
-AST Parsed     : 44000/44000 (100.00%)
-Positive Passed: 44000/44000 (100.00%)
-Negative Passed: 3915/3915 (100.00%)
-
-Babel Summary:
-AST Parsed     : 2065/2071 (99.71%)
-Positive Passed: 2062/2071 (99.57%)
-Negative Passed: 1332/1502 (88.68%)
-
-TypeScript Summary:
-AST Parsed     : 2337/2337 (100.00%)
-Positive Passed: 2331/2337 (99.74%)
-Negative Passed: 673/2535 (26.55%)
-```
 
-[Test262 conformance](https://github.com/tc39/test262) is complete. TypeScript parsing is complete.
+We also plan to port essential plugins such as [eslint-plugin-import] and [eslint-plugin-jest].
+
+#### 🏆 Linter Performance
+
+The linter is 50 - 100 times faster than [ESLint] depending on the number of cores used.
+It completes in less than a second for most codebases with a few hundred files and completes in a few seconds for
+larger monorepos. See [bench-javascript-linter](https://github.com/Boshen/bench-javascript-linter) for details.
+
+As an upside, the binary is approximately 3MB, whereas [ESLint] and its associated plugin dependencies can easily exceed 100.
+
+You may also download the linter binary from the [latest release tag](https://github.com/web-infra-dev/oxc/releases/latest) as a standalone binary,
+which means you can run the linter without a Node.js installation in your CI.
+
+<details>
+  <summary>How is it so fast?</summary>
+  <ul>
+    <li>Oxc parser is used</li>
+    <li>AST visit is fast operation due to linear memory scan from the memory arena</li>
+    <li>Files are linted in parallel, so scales with the total number of CPU cores</li>
```

Comment:
This is something you can do in Node.js too. Should you mention multi threading instead?

Reply:
This is a hard one, it's kind of hard to explain native multi-threading vs node.js parallellism? I changed this to "Files are linted in a multi-threaded environment".